content/docs/configure-destination/crowdstrike-falcon/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Sending messages to Falcon LogScale
+linktitle: Falcon LogScale
+---
+
+Starting with version 4.3.0, {{% param "product_name" %}} can send messages to [Falcon LogScale](https://library.humio.com/) using its [Ingest Structured Data API](https://library.humio.com/integrations/api-ingest.html#api-ingest-structured-data).
+
+## Prerequisites
+
+- Create an [Ingest token](https://library.humio.com/falcon-logscale-self-hosted/ingesting-data-tokens.html) for {{% param "product_name" %}} to use in the `token()` option of the destination. <!-- FIXME When creating the token, [set/do not set a parser](https://library.humio.com/data-analysis/parsers-assigning-to-ingest-tokens.html) for the token. -->
+
+## Ingest Structured Data API
+
+The `logscale()` destination feeds LogScale via the [Ingest Structured Data API](https://library.humio.com/integrations/api-ingest.html#api-ingest-structured-data).
+
+Minimal configuration:
+
+```sh
+destination d_logscale {
+  logscale(
+    token("your-logscale-ingest-token")
+  );
+};
+```
+
+This driver is actually a reusable configuration snippet configured to send log messages using the `http()` driver using a template. You can find the source of this configuration snippet on [GitHub](https://github.com/syslog-ng/syslog-ng/blob/master/scl/logscale/logscale.conf).
+
+## Options
```

Comment:
We could link to the `http()` dest options, which can be used for the `logscale()` dest, too: https://axoflow.com/docs/axosyslog-core/chapter-destinations/configuring-destinations-http-nonjava/reference-destination-http-nonjava/